en/docs/get-started/tutorials/create-your-first-event-triggered-using-containerized-app.md

```diff
@@ -0,0 +1,129 @@
+
+2. On the **Home** page, click **+Create Project**.
+
+3. Enter a unique name and a description for the project, and select a region.
+
+4. Click **Create**.  
+
+5. Go to the **Event-Triggered Integration** card, click **Create**, and enter the following information:
+
+    | **Field**       | **Value**               |
+    |-----------------|-------------------------|
+    | **Name**        | `MQTT listener`           |
+    | **Description** | `My sample listener`    |
+    
+6. Click **Next**.
+
+7. To allow Choreo to access your GitHub account, click **Authorize with GitHub**. 
+
+	!!! tip
+    	    If you have not already connected your GitHub repository to Choreo, enter your GitHub credentials, and select the [sample application](https://github.com/wso2/choreo-examples) GitHub repository to install the [Choreo GitHub App](https://github.com/marketplace/choreo-apps).
+
+8. In the **Connect Repository** dialog box, enter the following information:
+
+    | **Field**             | **Description**                                   |
+    |-----------------------|---------------------------------------------------|
+    | **GitHub Account**    | Your account                                      |
+    | **GitHub Repository** | `choreo-examples`                                 |
+    | **Branch**            | `main`                                            |
+    | **Build Preset**      | Click **Dockerfile**                        |
+    | **Path**              | `ipaas/containerized/containerized-rabbitmq-listener` |
+    | **Dockerfile Path**   | `Dockerfile` |  
+    | **Docker Context Path**   | `.` | 
+
+	!!! tip
+    	    - **Build Preset** specifies the type of build to run depending on the implementation of the component. It converts the integration code into a Docker image that can run on the Choreo cloud. If an integration is developed using [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/), select **Micro Integrator** as the build preset. If integration is developed using the [Ballerina language](https://ballerina.io), select **Ballerina** as the build preset. If integration is Containerized application, select **Dockerfile** as the build preset.
+
+            - **Path** specifies the location of the project to build the component. 
+
+9.   Click **Create**. This creates the component and takes you to the **Deploy** view, where you can proceed to deploy the component.
+
+
+## Step 2: Deploy
+Before you deploy the component, you must follow the steps given below to assign appropriate RabbitMQ credentials to the environment variables:
+
+1. In the **Deploy** view, go to the **Development** card and click **Configure via the DevOps Portal**. This takes you to the **Configs & Secrets** page in the **DevOps** profile, where you can apply environment-specific configuration files, environment variables, and other secret resources to containers.
```

Comment:
@ThusharaSampath I think we have to change this to match the new UI where we have the devops portal components in the deploy view.